java/wavefrontsender.md

```diff
@@ -0,0 +1,212 @@
+# WavefrontSender
+
+## Table of Content
+
+* [Overview](#Overview)
+* [Option 1: Sending Data via the Wavefront Proxy](#Option-1-Sending-Data-via-the-Wavefront-Proxy)
+* [Option 2: Sending Data via Direct ingestion](#Option-2-Sending-Data-via-Direct-ingestion)
+* [Share a WavefrontSender](#Share-a-WavefrontSender)
+* [What's next](#What's-next)
+
+## Overview
+
+You can choose to send metrics, histograms, or trace data from your application to the Wavefront service using one of the following techniques:
```

Comment:
You can send metrics, histograms, or trace data from your application to the Wavefront service using a Wavefront proxy or direct ingestions. 
(Then the 2 bullets. I think it's fine to give information right away)